prettier.config.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  semi: true,
+  printWidth: 100,
+  tabWidth: 2,
+  singleAttributePerLine: true,
```

Comment:
Amazing for readability

![image](https://github.com/user-attachments/assets/b2141b06-fe06-40bd-8808-829e3408697c)
![image](https://github.com/user-attachments/assets/2ec32008-468a-455c-b13d-adad505cabfc)